R/generate_fossils.R

```diff
@@ -39,6 +62,54 @@ png("assets/img/vertical_tree@1x.png", width = width, height = height)
 plot(tree, show.tip.label = FALSE, no.margin = TRUE)
 dev.off()
 
+rank <- "class"
+named_rank <- "hello"
+"class" <- c(get("class"), named_rank)
+
+
+class <- character(0)
+subclass <- character(0)
+infraclass <- character(0)
+megacohort <- character(0)
+supercohort <- character(0)
+cohort <- character(0)
+subcohort <- character(0)
+infracohort <- character(0)
+section <- character(0)
+subsection <- character(0)
+division <- character(0)
+subdivision <- character(0)
+series <- character(0)
+superorder <- character(0)
+order <- character(0)
+suborder <- character(0)
+infraorder <- character(0)
+family <- character(0)
```

Comment:
This should be stored as a list like above, and populated using the `wanted_ranks`, rather than a manually-coded sequence of variables.